migrations/20220424193304-create_users_table.sql

```diff
@@ -0,0 +1,16 @@
+
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS users
+(
+    id SERIAL,
```

Comment:
also any reason why not just to do ` id SERIAL NOT NULL PRIMARY KEY`

Reply:
done

Reply:
then you dont need this at all `CONSTRAINT users_pkey PRIMARY KEY (id)`